tool/jt.rb

```diff
@@ -29,6 +29,9 @@
 GRAAL_DIR = File.expand_path('../graal', TRUFFLERUBY_DIR)
 PROFILES_DIR = "#{TRUFFLERUBY_DIR}/profiles"
 
+CACHE_EXTRA_DIR = File.expand_path('~/.mx/cache/extra')
```

Comment:
Could you use `~/.mx/cache/truffleruby` or `~/.mx/cache/truffleruby-deps` to clarify what it's used by (and potential conflicts)?

Reply:
Done.